bigquery/google/cloud/bigquery/table.py

```diff
@@ -50,6 +50,8 @@
     '{:d} bytes remaining.')
 _DEFAULT_NUM_RETRIES = 6
 
+_FakeResponse = collections.namedtuple('_FakeResponse', ['status'])
```

Comment:
Worth either

- Making a note
- Filing an issue to delete this when `make_exception` gets refactored